backend/tracim_backend/lib/core/application.py

```diff
@@ -6,33 +6,53 @@
 from tracim_backend.app_models.workspace_menu_entries import all_content_menu_entry
 from tracim_backend.app_models.workspace_menu_entries import dashboard_menu_entry
 from tracim_backend.exceptions import AppDoesNotExist
+from tracim_backend.lib.utils.app import TracimApplication
+from tracim_backend.lib.utils.app import TracimContentType
 
 if typing.TYPE_CHECKING:
     from tracim_backend.models.data import Workspace
-    from tracim_backend.app_models.applications import Application  # noqa:F401
-    from tracim_backend.app_models.contents import ContentType  # noqa: F401
+    from tracim_backend.config import CFG
+    from tracim_backend.app_models.applications import TracimApplicationInContext  # noqa:F401
+    from tracim_backend.app_models.contents import ContentTypeInContext  # noqa: F401
 
 
 class ApplicationApi(object):
-    def __init__(self, app_list: List["Application"], show_all: bool = False) -> None:
+    def __init__(self, app_list: List["TracimApplication"], show_all: bool = False) -> None:
```

Comment:
Can you add a docstring about parameters ? (specially for `show_all`)

Reply:
renamed to show_inactive + doc.